core/valor_core/detection.py

```diff
@@ -643,6 +621,21 @@ def _calculate_detailed_pr_metrics(
     ) or (detailed_pr_joint_df is None):
         return []
 
+    if _check_if_series_contains_masks(
+        detailed_pr_joint_df.loc[
+            detailed_pr_joint_df["converted_geometry_pd"].notnull(),
```

Comment:
Should this be a gt check it looks like this statement would result in `pd or pd`